ui/components/MesheryPerformance/PerformanceProfiles.js

```diff
@@ -146,23 +152,47 @@ function PerformanceProfile({ updateProgress, enqueueSnackbar, closeSnackbar })
     return function (error) {
       updateProgress({ showProgress : false });
 
-      enqueueSnackbar(`${msg}: ${error}`, { variant : "error",
+      enqueueSnackbar(`${msg}: ${error}`, {
+        variant : "error",
         action : function Action(key) {
           return (
             <IconButton key="close" aria-label="Close" color="inherit" onClick={() => closeSnackbar(key)}>
               <CloseIcon />
             </IconButton>
           );
         },
-        autoHideDuration : 8000, });
+        autoHideDuration : 8000,
+      });
     };
   }
 
   return (
     <>
       <div style={{ padding : "0.5rem" }}>
-        <div style={{ margin : "0 0 2rem auto", width : "fit-content" }}>
-          <ViewSwitch view={viewType} changeView={setViewType} />
+        <div style={{ margin : "0 0 2rem auto", display : "flex", justifyContent : "space-between", paddingLeft : "1rem" }}>
+
+          {testProfiles.length ? (
+            <div style={{ width : "fit-content", alignSelf : "flex-start" }}>
+              <Button
+                aria-label="Add Performance Profile"
+                variant="contained"
+                color="primary"
+                size="large"
+                // @ts-ignore
+                onClick={() => setProfileForModal({})}
+              >
+                <AddIcon />
+                Add Performance Profile
+              </Button>
+
+            </div>
+
+          ) : (
+            ''
+          )}

```

Comment:
I think there is no need of extra linebreaks, lying here and there. you can delete those. Also you can use && statement to even get more simplified code look.
like 
```
testProfiles.length  && (....the actual code when the length of test profile is zero, here that div portion
```